### ISSUE TITLE
stylelint shows CssSyntaxError to a JPG file

### ISSUE THREAD

**Issue by benebsiny** [stylelint/stylelint#4406]
> Clearly describe the bug

When I added a JPG file to my workspace, then stylelint showed me an error message that is 
```
src/web/img/Taiwan.jpg
12:87  ×  Unclosed bracket   CssSyntaxError
```

> Which rule, if any, is the bug related to?

I'm not sure.

> What CSS is needed to reproduce the bug?

No, it's just a JPG file.

> What stylelint configuration is needed to reproduce the bug?

```json
{
    "extends": "stylelint-config-standard",
    "rules": {
        "declaration-block-trailing-semicolon": null
    }
}
```

> Which version of stylelint are you using?

`11.1.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

e.g. "CLI with `stylelint ./src/web/`"

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No, it's just a JPG file.

> What did you expect to happen?

No error to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following error were flagged:

```shell
src/web/img/Taiwan.jpg
 12:87  ×  Unclosed bracket   CssSyntaxError
```

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

You need to specify pattern to match files as input. I believe stylelint can't receive just a folder. E. g. the following will lint all `.css` files in `./src/web/` and all nested folders:

```
stylelint "./src/web/**/*.css"
```

**Comment by benebsiny** [comment]
It works!! Thank you!!!
